app/js/controllers/main-ctrl.js

```diff
@@ -2,8 +2,139 @@
 
 angular
 .module('app.controllers')
-.controller('MainCtrl', function($scope, apiDescriptor) {
+.controller('MainCtrl', function($scope, apiDescriptor, Restangular) {
 	apiDescriptor.then(function(apiDescription) {
 		$scope.apidesc = apiDescription.data;
+		//console.log(preProcess.loadCurrentEBoard());
```

Comment:
remove `console.log`. keep the extra line though

Reply:
Fixed